packages/config/index.js

```diff
@@ -176,7 +176,7 @@ class ConfigManager extends EventEmitter {
       validate: (schema, path, parentSchema, data) => {
         const resolved = resolve(this.dirname, path)
         data.parentData[data.parentDataProperty] = resolved
-        return true
+        return typeof path === 'string' && path.trim() !== ''
```

Comment:
I would check it at the beginning of the function before modifying the data. But it's not critical.

Reply:
Want to make this change or is fine for now?

Reply:
it would be nice if you can, but it can be merged without it

Reply:
I will make the changes now!!